Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -300,10 +301,120 @@ namespace AzFramework
     ////////////////////////////////////////////////////////////////////////////////////////////////
     uint32_t XcbNativeWindow::GetDisplayRefreshRate() const
     {
-        // [GFX TODO][GHI - 2678]
-        // Using 60 for now until proper support is added
+        xcb_generic_error_t* error = nullptr;
+
+        xcb_screen_iterator_t iter = xcb_setup_roots_iterator(xcb_get_setup(m_xcbConnection));
+        xcb_screen_t* screen = iter.data;
+
+        const xcb_query_extension_reply_t* extReply = xcb_get_extension_data(m_xcbConnection, &xcb_randr_id);
+        if (!extReply || !extReply->present)
+        {
+            AZ_Warning(XcbErrorWindow, extReply != nullptr, "Failed to get extension RandR.");
+            return 60;
```

Comment:
(related to above comment), lets make 60 a constexpr constant since its used again later.